emsdk.py

```diff
@@ -294,11 +294,11 @@ def remove_readonly_and_try_again(func, path, exc_info):
 def import_pywin32():
   if WINDOWS:
     try:
-      import win32api
-      import win32con
-      return win32api, win32con
-    except Exception:
-      exit_with_error('Failed to import Python Windows extensions win32api and win32con. Make sure you are using the version of python available in emsdk, or install PyWin extensions to the distribution of Python you are attempting to use. (This script was launched in python instance from "' + sys.executable + '")')
+        import winreg
+    except ImportError:
+        import _winreg as winreg
+    import ctypes
+    return winreg, ctypes
```

Comment:
Given that these are builtin module can we just do this at the top level no?   No need to delay the import I think.

Reply:
Lazy loading will still have performance benefits. Python is not intelligent enough to no spend time importing these when they are not used.

Reply:
I don't think we need to worry about the performance penalty of loading winreg and ctypes in this script.

I believe these imports were imported late like this we could report an error and handle them being missing.   Now that can't happen which is great, so there is no need to delay the importing IMHO.

Keeping all imports at the top level helps make the program more readable IMHO.

Reply:
Here you go: https://github.com/emscripten-core/emsdk/pull/635/commits/1590416a9a08f6028a0068b9e27897d947294e7b

Reply:
Alternatively maybe just inline these imports the top of `win_set_environment_variable_direct` before the `try` block?

And you can drop the `if WINDOWS` too since `win_set_environment_variable_direct` is already windows only?